heclib/heclib_c/src/headers/fortran_string_len_size.h

```diff
@@ -0,0 +1,23 @@
+#ifndef __FORTRAN_STRING_LEN_SIZE_INCLUDED__
+#define __FORTRAN_STRING_LEN_SIZE_INCLUDED__
```

Comment:
Are these basically enforced by the compilers, we can't just use "size_t" for everything?

Needs more explanation for future readers.

Reply:
Also this might look cleaner if you duplicated the logic (defined _MSC_VER && defined(_WIN64) vs the nesting.

Reply:
Turns out we *can* now use size_t for the Fortran string length hidden variables for all the compilers we use. That wasn't always the case.

Reply:
Always nice when things actually standardize.